### DISCUSSION TITLE
HA Core 2023.2.0 break the Solax integration for anyone else?

### DISCUSSION THREAD

**Discussion by ezfrag2021** [wills106/homeassistant-solax-modbus#319] [Q&A]
I installed HA Core update last night and Solax failed to load. 

Logger: homeassistant.config_entries
Source: config_entries.py:816
First occurred: 15:27:05 (1 occurrences)
Last logged: 15:27:05
Error occurred loading configuration flow for integration solax_modbus: No module named 'pymodbus.client.sync'

Rolled back to the pre-update backup and it's working again. Anyone else having problems?

No updates showing for Solax.

**Comment by infradom** [comment]
you might be running a very old version of this integration. The latest version of the integration  should be compatible (I just upgraded to HA core 2023.2 without this problem

**Reply by ezfrag2021** [reply]
I don't have any prompts within HA to update to a new version so I assume it's up-to-date.

How can I check the version I'm running?

**Reply by wills106** [reply]
How did you initially install this? Manually or through HACS?

If you look in manifest.json you will see it.
Sounds like you are on a really old version of the Integration such as 0.5.x or 0.6.x possibly?

**Reply by ezfrag2021** [reply]
{
  "domain": "solax_modbus",
  "name": "SolaX Inverter Modbus",
  "version": "0.5.6",
  "documentation": "https://github.com/wills106/homsassistant-solax-modbus",
  "issue_tracker": "https://github.com/wills106/homsassistant-solax-modbus/issues",
  "requirements": ["pymodbus>=2.5.3"],
  "dependencies": [],
  "codeowners": ["@wills106"],
  "config_flow": true,
  "iot_class": "local_polling"
}

Looks like you were both correct. I always thought it was installed via HACS but possibly I installed it before it was available via HACS? I didn't see an update available so just assumed it was up-to-date. I will delete the integration and install via HACS instead.

Appreciate the help here.
